src/main/target/WORMFC/target.h

```diff
@@ -86,34 +139,63 @@
 #define UART6_RX_PIN            PC7
 #define UART6_TX_PIN            PC6
 
+#if defined(PIRXF4)
+#define SERIAL_PORT_COUNT       6 //VCP, USART1, USART2, USART3, USART4, USART6
+#else
 #define SERIAL_PORT_COUNT       5 //VCP, USART1, USART3, USART4, USART6
+#endif
 
 #define USE_ESCSERIAL
 #define ESCSERIAL_TIMER_TX_PIN  PB9
 
+//GPS ----------------------------------------------
+#define USE_GPS
+#define USE_GPS_UBLOX
+#define USE_GPS_NMEA
+
+//GPS ----------------------------------------------
```

Comment:
Duplicate lines for GPS?

Reply:
Will fix everything tomorrow, sorry a lot of busy days...

Reply:
All good, no hurry. Marking pull requests where changes are requested as such helps me keep tabs on the list of open pull requests.